README.md

```diff
@@ -17,6 +17,19 @@ cross-platform way. Initially, it will provide a streamlined experience for deve
 * [Getting started with web-ext](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Getting_started_with_web-ext)
 * [Command reference](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/web-ext_command_reference)
 
+Here are the commands you can run. Click on each one for detailed documentation or use `--help` on the command line, such as `web-ext --help build`.
```

Comment:
nit, not a big deal (and I'm pretty sure that with yargs the order doesn't matter), but as a command line user I'm more used to see the opposite `web-ext build --help`

Reply:
Good call. That's more intuitive too since you may start typing `web-ext run ...` and then realize you don't know the name of some option.